index.html

```diff
@@ -54,28 +54,43 @@
     }"
   >
     <nav
-      class="flex flex-wrap justify-between items-center px-4 py-2 gap-2 sticky top-0"
+      class="flex items-center px-4 py-2 gap-2 sticky top-0"
       :class="{ 'z-10': characterMenuOpen }"
     >
-      <button
-        onClick="setActiveRoom(null); render();"
-        class="rounded-md border border-transparent bg-slate-600 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-slate-700 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2"
-      >
-        Back
-      </button>
-      <h1 id="title" class="text-xl font-bold mx-auto truncate max-w-[150px]">
-        Omnispace
-      </h1>
-      <button
-        onClick="logout()"
-        class="rounded-md border border-transparent bg-slate-600 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-slate-700 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2"
-      >
-        Logout
-      </button>
-
-      <!-- Start customize character sidebar -->
-      <div
-        x-data="{
+      <div class="flex-1">
+        <button
+          id="back"
+          onClick="setActiveRoom(null); render();"
+          class="rounded-md border border-transparent bg-slate-600 py-2 px-2 text-sm font-medium text-white shadow-sm hover:bg-slate-700 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2 invisible min-w-[38px]"
+        >
+          <svg
+            fill="none"
+            stroke="currentColor"
+            stroke-width="1.5"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+            aria-hidden="true"
+            class="h-5 w-5"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18"
+            ></path>
+          </svg>
+        </button>
+      </div>
+
+      <div class="min-w-0 flex-1 text-center">
+        <h1 id="title" class="text-xl font-bold mx-auto truncate w-full">
+          All Chats
+        </h1>
+      </div>
```

Comment:
is there a reason this is enclosed in another `<div>`? I believe `<h1>` elems are block elems so you can just apply the classes to it directly?

Reply:
For truncate without fixed width to work you must set its width to 100% and its parent maxWidth to 0

Reply:
same goes for the `<button>`s

Reply:
And for buttons I think in order to make left and right elements same width we need a parent div